### PR TITLE
Add Event Hubs and Functions to index

### PR DESCRIPTION
**Pull request by mmacy**

### CODE REVIEW

File: docs-ref-conceptual/index.yml [hunk 1]
```diff
@@ -45,34 +45,39 @@ sections:
...
       href: https://docs.microsoft.com/azure/aks/kubernetes-walkthrough?toc=%2Fpython%2Fazure%2FTOC.json
-    - image: 
-        src: https://docs.microsoft.com/en-us/azure/virtual-machines/media/index/virtual-machines.svg
+    - image:
+        src: https://docs.microsoft.com/azure/virtual-machines/media/index/virtual-machines.svg
       title: Virtual Machines
       html: <p>Use a Linux distribution or Windows Server to run your code at scale</p>
       href: https://docs.microsoft.com//azure/virtual-machines/linux/quick-create-cli?toc=%2Fpython%2Fazure%2FTOC.json
+    - image:
+        src: https://docs.microsoft.com/azure/virtual-machines/media/index/AzureFunctions.svg.svg
```
Comment (ggailey777) [inline docs-ref-conceptual/index.yml]: .svg.svg might be a problem 😉

Comment (mmacy) [reply docs-ref-conceptual/index.yml]: omgnoooooooo

Comment (mmacy) [reply docs-ref-conceptual/index.yml]: Ugh. Thanks for the catch, @ggailey777. That link was wrong in all sorts of ways. :cry:

**Comment by ggailey777** [comment]
@mmacy does this build get staged anywhere?

**Comment by mmacy** [comment]
@ggailey777 Looks like there is no staging for this repo. 

@dend What would be involved in enabling staging for this repo?

**Comment by mmacy** [comment]
Hey @dend, has the enabling of staging change not made it into live, or is there something else that might need to be done to enable staging?

**Comment by mmacy** [comment]
#sign-off

@arob98 this'll require your authorization, I believe. This update adds Functions and Event Hubs to the landing page for Python SDKs.

Cc: @rloutlaw

File: docs-ref-conceptual/index.yml [hunk 2]
```diff
@@ -45,34 +45,39 @@ sections:
...
-        src: https://docs.microsoft.com/en-us/azure/virtual-machines/media/index/virtual-machines.svg
+    - image:
+        src: https://docs.microsoft.com/azure/virtual-machines/media/index/virtual-machines.svg
       title: Virtual Machines
       html: <p>Use a Linux distribution or Windows Server to run your code at scale</p>
       href: https://docs.microsoft.com//azure/virtual-machines/linux/quick-create-cli?toc=%2Fpython%2Fazure%2FTOC.json
+    - image:
+        src: https://docs.microsoft.com/azure/media/index/AzureFunctions.svg
+      title: Functions
+      html: <p>Run code on-demand without having to explicitly provision or manage infrastructure</p>
```
[the remark this answers is not in the document]
Comment (mmacy) [reply docs-ref-conceptual/index.yml]: @asavaritayal Just so I'm clear on the request, are you requesting we replace this:

`Run code on-demand without having to explicitly provision or manage infrastructure`

...with this:

`Create a serverless Python function`

[the remark this answers is not in the document]
Comment (asavaritayal) [reply docs-ref-conceptual/index.yml]: Yes, that's the suggestion but I'm open to discussion.

[the remark this answers is not in the document]
Comment (mmacy) [reply docs-ref-conceptual/index.yml]: Sounds good to me, just making sure I had it right. :smile: 

Will update, stand by...

#hold-off

**Comment by mmacy** [comment]
#sign-off

Approval request: @arob98 @rloutlaw

File: docs-ref-conceptual/index.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmacy) [reply docs-ref-conceptual/index.yml]: Hi @asavaritayal, this change has been made.

**Comment by mmacy** [comment]
@Taojunshen Please merge, thanks!

**Pull request merged**